migrations/20170210150717_create-table-userdata.js

```diff
@@ -4,7 +4,7 @@ exports.up = function(knex, Promise) {
     table.increments('id').primary()
     table.string('username').notNullable()
     table.string('inst_id').notNullable()
-    table.string('token').notNullable()
+    table.string('token', 1000).notNullable()
```

Comment:
The `text` column type is unbounded and compressed by default. You might like that.